core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -25,6 +25,7 @@ import java.util.zip.{ZipInputStream, ZipOutputStream}
 
 import scala.collection.JavaConverters._
 import scala.concurrent.duration._
+import scala.runtime.BoxesRunTime
```

Comment:
Not sure why this needed? Tests show the same result with and without it.

Reply:
Oh , I see now. This was coming from a previous code:
`assert(BoxesRunTime.unboxToLong(opt.get) === expected.get)`